packages/fxa-content-server/app/scripts/views/settings/delete_account.js

```diff
@@ -32,7 +32,7 @@ var View = FormView.extend({
         notifier: options.notifier,
       });
     }
-    this._activeSubscriptions = [];
+    this._activeSubscriptions = [] || options.activeSubscriptions;
```

Comment:
Assertions added in cc087c5d2fd431549c8d617145db03184e6cea84 pass in mocked `activeSubscriptions`.